cloudferrylib/os/actions/get_filter.py

```diff
@@ -19,12 +19,14 @@
 class GetFilter(action.Action):
 
     def run(self, **kwargs):
-        search_opts = None
+        search_opts, search_opts_img = None, {}
         filter_path = self.cfg.migrate.filter_path
         if utl.read_yaml_file(filter_path):
             filter_config = utl.read_yaml_file(filter_path)
             if utl.INSTANCES_TYPE in filter_config:
                 search_opts = filter_config[utl.INSTANCES_TYPE]
+            search_opts_img = filter_config[utl.IMAGES_TYPE]
         return {
-            'search_opts': search_opts
+            'search_opts': search_opts,
+            'search_opts_img': search_opts_img
```

Comment:
I don't see this used anywhere else, did you forget to remove `search_opts_img` or is it me reading it wrong?..